packages/create-pantheon-decoupled-kit/src/generators/next-drupal-umami-addon.generator.ts

```diff
@@ -11,7 +22,16 @@ export const nextDrupalUmamiAddon: DecoupledKitGenerator = {
 			message: 'Where should the output go?',
 			default: `${process.cwd()}/umami-demo`,
 		},
+		{
+			name: 'tailwindcss',
+			message: 'Would you like to include tailwindcss?',
+			type: 'confirm',
+			default: true,
+		},
 	],
+	data: {
+		drupal: true,
```

Comment:
Doesn't necessarily need to change now, but thinking ahead to being able to 'filter' for specific cms add-ons from something like the terminus plugin, I wonder if this data would be easier to work with in a dedicated field like 'cmsType' rather than a variety of CMS specific booleans.